docs/source/install/upgrades.rst

```diff
@@ -167,6 +167,102 @@ The following sections call out the migration scripts that need to be run when u
 respective version. If you are upgrading across multiple versions, make sure you run the scripts for
 any skipped versions:
 
+
+v3.3
+''''
+
+* MongoDB 4.0 is the new default version. Previously the version of MongoDB was 3.4 and
+  the supported upgrade path to MongoDB 4.0 is ``3.4 -> 3.6 -> 4.0``.
+  Official documentation on how to upgrade MongoDB can be found here:
+  * https://docs.mongodb.com/manual/release-notes/3.6-upgrade-standalone/
+  * https://docs.mongodb.com/manual/release-notes/4.0-upgrade-standalone/
+
+  A summary of the steps to take is outlined below assuming you will be migrating
+  through the path ``3.4 -> 3.6 -> 4.0``
+
+  Ubuntu:
+
+  .. sourcecode:: bash
+
+     # Stop StackStorm
+     sudo st2ctl stop
+
+     # Ensure current MongoDB feature compatability level is set to 3.4
+     mongo admin --username admin --password Password --quiet --eval "db.adminCommand( { setFeatureCompatibilityVersion: '3.4' } )"
+
+     # Upgrade MongoDB packages to 3.6
+     # TODO how do we update the apt source?
+     sudo apt-get -y clean
+     sudo apt-get -y update
+     sudo apt-get -y install mongodb-*
+
+     # Set MongoDB feature compatability level to 3.6
+     mongo admin --username admin --password Password --quiet --eval "db.adminCommand( { setFeatureCompatibilityVersion: '3.6' } )"
+
+     # TODO Upgrade MongoDB packages to 4.0
+     # TODO how do we update the apt source?
+     sudo apt-get -y clean
+     sudo apt-get -y update
+     sudo apt-get -y install mongodb-*
+
+     # Set MongoDB feature compatability level to 4.0
+     mongo admin --username admin --password Password --quiet --eval "db.adminCommand( { setFeatureCompatibilityVersion: '4.0' } )"
+
+     # Start StackStorm
+     sudo st2ctl start
+
+
+  RHEL/CentOS:
```

Comment:
Can we add a comment to say only relevant for RHEL/CentOS 7.x (as for 8.x) we were already at mongodb 4.0.